src/post/post.entity.ts

```diff
@@ -221,6 +228,19 @@ export class Post extends BaseEntity {
         };
     }
 
+    addMention(account: Account) {
+        if (!account.id) {
```

Comment:
This case is not possible

Reply:
Removed the check here https://github.com/TryGhost/ActivityPub/pull/666/commits/4adb4d63b88e42c797ef80a882cf288e5f96fa1c#diff-3522562e572113f90d24c5500be2ae5bf5647a64867dfb920f6727a32a18facfL232